core/src/main/java/edu/ucr/cs/riple/core/metadata/graph/Node.java

```diff
@@ -59,7 +59,7 @@ public class Node {
   public Set<Fix> triggeredFixes;
 
   /** Collection of triggered errors if tree is applied. */
-  public ImmutableSet<Error> triggeredErrors;
+  public ImmutableList<Error> triggeredErrors;
```

Comment:
Yes, we can have multiple errors with same message nonNullTarget and type.

Reply:
Um, can you give me an example here of distinct errors with the same `messageType`, `message`, and `nonnullTarget` that we want to keep track of? Also, any reason we can't simply refine what `hashCode()` takes into account so that distinct errors mean distinct elements in this list/set? 

I still don't understand what changes when we change how we track parameter fixes coming from downstream deps that makes us now want to consider repetitions but not before (if this is a separate bug fix from the core change of this PR, then it being mixed together with this feature makes it harder to understand).

Reply:
Discussed offline, we probably want to track an issue here to switch back to a set once we have enough position info in `errors.tsv` to accurately distinguish separate errors in the same region with the same `messageType` and `nonnullTarget`, but for now this is not blocking 🙂

Reply:
Please find the issue here #67.